server/db/db.js

```diff
@@ -328,7 +328,7 @@ async function getLocations(pgClient) {
       WHERE d.device_type = $1
       ORDER BY c.display_name, d.display_name
       `,
-      [DEVICE_TYPE.DEVICE_SENSOR],
+      [DEVICE_TYPE.DEVICE_SENSOR_SINGLESTALL],
```

Comment:
this function (and many others i think?) wouldn't work with singlestall only as device type. u need to do both singlestall and multistall, especially since the functions u changed are for dasboard stuff.

Reply:
Yup, you're right. I will change all the database functions to handle and retrieve multistall devices as well. Along with that, with each corresponding function, I can change the test cases to use handle input for both singlestall and multistall. This would probably involve writing new test cases. Thanks!

Reply:
Completed in latest commits